legend-engine-pure-code-compiled-core/src/main/resources/core/pure/executionPlan/platformBinding/typeInfo/typeInfo.pure

```diff
@@ -649,3 +656,89 @@ function  <<access.private>> meta::pure::executionPlan::platformBinding::typeInf
+                                          $fe.func->match([
+                                                            p:Property<Nil,Any|*>[1]|print(if($debug.debug,|$debug.space+'  property:'+$p.name->toOne()+'\n', |''));
+                                                                                     let leftSide = $params->at(0)->scanTypesFromAny($processed,$debug->indent());
+                                                                                     ,
+                                                            q:QualifiedProperty<Any>[1]|print(if($debug.debug,|$debug.space+'  qualifier:'+$q.name->toOne()+'\n', |''));
+                                                                                        let leftSide = $params->at(0)->scanTypesFromValueSpecification($processed->concatenate($q), $debug->indent());
+                                                                                        let fTypes= $q->scanTypesFromFunction($processed, $debug->indent());
+                                                                                        $leftSide->concatenate($fTypes);,
+                                                            z:Function<Any>[1]|print(if($debug.debug,|$debug.space+'  function: \''+$fe.func.name->toOne()+'\'\n',|''));
+                                                                    let pp = $params->map(v|$v->scanTypesFromAny($processed->concatenate($z),$debug->indent()));
+                                                                    let fTypes= $z->scanTypesFromFunction($processed, $debug->indent());
+                                                                    $pp->concatenate($fTypes);
+                                                                          
+                                                          ]);,
+               i:InstanceValue[1] | print(if($debug.debug,|$debug.space+'Processing Instance'+  + $i->printValueSpecification('') + '\n',|''));
+                                    $i.values->evaluateAndDeactivate()->map( v | $v->scanTypesFromAny($processed, $debug->indent()));
+                                    ,
+               ve:VariableExpression[1] | print(if($debug.debug,|$debug.space+'Processing Variable '+$ve->printValueSpecification('') +'\n',|''));
+                                         $ve.genericType.rawType;,
+               ervs:ExtendedRoutedValueSpecification[1] | print(if($debug.debug,|$debug.space+'Bypassing ExtendedRoutedValueSpecification\n',|''));
+                                                          $ervs.value->scanTypesFromValueSpecification($processed, $debug->indent());,
+               frvs:FunctionRoutedValueSpecification[1] | print(if($debug.debug,|$debug.space+'Bypassing FunctionRoutedValueSpecification\n',|''));
+                                                          $frvs.value->scanTypesFromValueSpecification($processed, $debug->indent());
+             ]
```

Comment:
Can we add a default case to the match function avoid failures?

Reply:
added